docs/software/installing/easybuild.md

```diff
@@ -1,277 +1,292 @@
+EasyBuild repository"](#building-your-own-easybuild-repository). If you want
+more information about the full configuration of EasyBuild, you can execute
+
 ```bash
-eb --show-config
+$ eb --show-config
 ```
 
 Now an EasyBuild build recipe is a file with a name that consists of different
 components. Consider, e.g., the build recipe
-```
+
+```text
 GROMACS-2021.4-cpeGNU-22.06-PLUMED-2.7.4-CPU.eb
 ```
+
 The first part of the name, `GROMACS`, is the name of the package. The second
 part of the name, `2021.4` is the version of GROMACS, in this case the initial
 2021 release. The next part, `cpeGNU-22.06`, denotes the so-called *toolchain*
 used for the build. The `cpeGNU` toolchain uses the `PrgEnv-gnu` programming
 environment, the `cpeCray` toolchain the `PrgEnv-cray` PE, the `cpeAOCC`
 toolchain the `PrgEnv-aocc` environment and the `cpeAMD` toolchain the
-`PrgEnv-amd` environment. 
-The version of the toolchain should
-match the version of the LUMI software stack or the installation will fail.
-(In fact, it is not just the version in the file name that should match but
-the version of the toolchain that is used in the recipe file.) The next part
-of the name, `-PLUMED-2.7.4-CPU`, is called the version suffix. Version suffixes
-are typically used to distinguish different builds of the same version of the
+`PrgEnv-amd` environment. The version of the toolchain should match the version
+of the LUMI software stack or the installation will fail. (In fact, it is not
+just the version in the file name that should match but the version of the
+toolchain that is used in the recipe file.) The next part of the name,
+`-PLUMED-2.7.4-CPU`, is called the version suffix. Version suffixes are
+typically used to distinguish different builds of the same version of the
 package. In this case, it indicates that it is a build of the 2021.4 version
-purely for CPU and also includes PLUMED as we have also builds without
-PLUMED (which is not compatible with every GROMACS version).
-
-EasyBuild is configured so that it searches in the user
-repository and two repositories on the system. The current directory is not
-part of the default search path but is easily added with a command line
-option. By default, EasyBuild will not install dependencies of a package
-and fail instead if one or more of the dependencies cannot be found, but that
-is also easily changed on the command line.
-If all needed EasyBuild
-recipes are in one of those repository or in the current
-directory, all you need to do to install the package is to run
+purely for CPU and also includes PLUMED as we have also builds without PLUMED
+(which is not compatible with every GROMACS version).
+
+EasyBuild is configured so that it searches in the user repository and two
+repositories on the system. The current directory is not part of the default
+search path but is easily added with a command line option. By default,
+EasyBuild will not install dependencies of a package and fail instead if one or
```

Comment:
Something to check about this and following lines...

Reply:
Do you have any suggestions for changes to these lines? I think the content is correct, though it probably could be presented in a more clear way.

Reply:
Added a comma that I think was needed. You can check it the sentences make sense still (from my point of view the comma belongs to that place)